client/innernet@.service

```diff
@@ -3,11 +3,17 @@ Description=innernet client daemon for %I
 After=network-online.target nss-lookup.target
 Wants=network-online.target nss-lookup.target
 PartOf=innernet.target
+# Disable systemd's unit start rate limiting logic, which could override Restart=always.
+# See https://unix.stackexchange.com/q/289629/352972
+StartLimitIntervalSec=0
 
 [Service]
 Type=simple
 ExecStart=/usr/bin/innernet up %i --daemon --interval 60
 Restart=always
+# When the daemon exits, wait this amount of secs before restarting. Prevents innernet from
+# start-looping each 100ms for example when there is a problem reaching the server.
+RestartSec=60
```

Comment:
If it's ok with you, let's make this 10 rather than 60 since it may be a temporary issue and we should be retrying harder if a failure has caused innernet to not fetch peers and update the interface.